scripts/build/scalingo.sh

```diff
@@ -11,8 +11,7 @@ if echo "$APP" | grep -o 'api'; then
     npx nx run-many -t build --projects=api,cron,tag:backend:queues
 
     echo "Running migrate for $APP"
-    # TODO: activate once the initial migration has been applied
-    # npx prisma migrate dev
+    npx prisma migrate dev
```

Comment:
```suggestion
    npx prisma migrate deploy
```

Pour ref: https://www.prisma.io/docs/orm/reference/prisma-cli-reference#migrate-deploy